Modules/Core/Common/test/itkShapedNeighborhoodIteratorTest.cxx

```diff
@@ -191,6 +192,21 @@ itkShapedNeighborhoodIteratorTest(int, char *[])
   }
   std::cout << it.GetPixel(off) << std::endl;
 
+  println("Testing interoperability with FlatStructuringElement");
+  using SE2Type = itk::FlatStructuringElement<4>;
+  SE2Type::RadiusType r;
+  r.Fill(2);
+  SE2Type k;
+  k = SE2Type::Cross(r);
+  itk::ShapedNeighborhoodIterator<TestImageType> iterator(r, img, img->GetBufferedRegion());
+  iterator.CreateActiveListFromNeighborhood(k); // this now compiles
```

Comment:
The comment "this now compiles" is meaningful now, in the context of this pull request, but it may no longer be understood in a few years. Personally I would just drop it. But you could also elaborate a bit on why this compiles (now).